community/basic_examples/hw_aware_synthesis.ipynb

```diff
@@ -0,0 +1,601 @@
+{
```

Comment:
<p>As a convention, there is no need to put the installation and authentication within the notebook (it makes it cleaner)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>